README.md

```diff
@@ -46,15 +46,15 @@ Thanks to our modular pricing architecture, new token primitives and new token s
 
 ![Modular pricing](https://i.ibb.co/kyWXZ6C/Arca-Fi-pricing.png)
 
-## Pricing logic
+## Pricing module
```

Comment:
This occurrence and the next two are the only exceptions where we can keep Pricing logic (because we are explaining here that our Pricing logic is modular, hence we call it pricing modules). :D